test/container_test.py

```diff
@@ -111,7 +217,7 @@ def test_set_volume_unit_conv(self):
         c.well(1).set_volume(".1:milliliter")
         self.assertTrue(c.well(1).volume == Unit(100, "microliter"))
         with self.assertRaises(ValueError):
-            c.well(2).set_volume("1:liter")
+            c.well(2).set_volume("1:milliliter")
```

Comment:
I wasn't sure whether to prefer the context-manager or method invocation of `assertRaises`; both were already present and I ended up using either arbitrarily. Do you have a preference?